### ISSUE TITLE
Issue with i18n: 404 Error When Accessing English Version of Site

### ISSUE THREAD

**Issue by EpsilonOrionis** [aralroca/next-translate#1140]
### Description

I've been experiencing an issue with internationalization using Next.js 13 and the next-translate package. The issue is that when I try to access the English version of my website by appending `/en` to the URL, I get a 404 error.

### Steps to Reproduce

1. Navigate to the default Italian version of the site (e.g., `http://<domain>`).
2. Try to switch to the English version by navigating to `http://<domain>/en`.

### Expected Behavior

The English version of the site should be displayed.

### Actual Behavior

I receive a 404 error.

### Configurations

My `next.config.js` is configured as follows:

```javascript
/** @type {import('next').NextConfig} */
const nextConfig = {
};

const nextTranslate = require('next-translate-plugin');

// Utilizza nextTranslate per estendere la configurazione
module.exports = nextTranslate(nextConfig);
```

And my `i18n.json`

```json
{
  "locales": ["en", "it"],
  "defaultLocale": "en",
  "pages": {
    "*": ["common"],
    "/": ["home", "example"]
  }
}
```

I have tried various troubleshooting steps, including verifying the setup of i18n.json, restarting the development environment, and even using experimental configurations in next.config.js, but the issue persists.

**Comment by michalstrzelecki** [comment]
I have been experiencing the same problem, but with `next-translate` 1.6. Latest working `nextjs` version is 13.4.9.

**Comment by michalstrzelecki** [comment]
I think that my issue is related to https://github.com/aralroca/next-translate/issues/1131. When I delete middleware file everything works again. 
`nextjs` version 13.4.19

**Comment by oscarrc** [comment]
@EpsilonOrionis I think it is related. Try to set i18n to undefined before exporting next.config  because if that is present Next.js normalizes the url by removing the locale (which makes sense when using built-in i18n with pages router but not with app router) hence the 404 error.

**Comment by boredland** [comment]
not having a middleware, removing i18n from the next.config helps for the development mode, but for me the production builds still lead to a 404.

**Comment by deifos** [comment]
Did you figure this out? 
I had the same issue and setting nextconfig like so.

`const nextConfig = {};
const nextTranslate = require("next-translate-plugin");
const config = nextTranslate(nextConfig) 
config.i18n =  undefined; 

module.exports = config;`

fixed the problem but then I do not get the locale on the request.nextUrl.locale in the middleware.

**Comment by aralroca** [comment]
This is solved in [3.0.0-canary.1](https://github.com/aralroca/next-translate/releases/tag/3.0.0-canary.1) version (both, next-translate & next-translate-plugin). Thanks

**Comment by iamvinny** [comment]
> This is solved in [3.0.0-canary.1](https://github.com/aralroca/next-translate/releases/tag/3.0.0-canary.1) version (both, next-translate & next-translate-plugin). Thanks

Doesn't seem to work for me on the canary version, is there any workaround for the current stable version?
